org.gnupg.GPA.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.gnupg.GPA
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: gpa
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  #- --filesystem=~/.gnupg:rw
+  #- --filesystem=xdg-run/gnupg:rw
+  - --filesystem=host
```

Comment:
host remains a broad permission

Reply:
You should be able to open any arbitrary file in GPA to crypt/decrypt/sign it, how else could this be achieved?

Reply:
It's not possible with GTK2 app.